pkgs/by-name/sk/skia/package.nix

```
@@ -62,7 +62,16 @@ stdenv.mkDerivation (finalAttrs: {
     vulkan-memory-allocator
   ];
 
-  configurePhase = ''
+  configurePhase = let
+    cpu = platform:
+      let name = platform.parsed.cpu.name;
+      in ({ "x86_64" = "x64";
+            "i686" = "x86";
+            "arm" = "arm";
+            "aarch64" = "arm64";
+          }.${platform.parsed.cpu.name}
+        or (throw "no entry for cpu: ${name}"));
```

We can inline stdenv.HostPlatform since it's the only actual parameter.
I'd also inline `platform.parsed.cpu.name`.

```suggestion
    cpu =
         ({ "x86_64" = "x64";
            "i686" = "x86";
            "arm" = "arm";
            "aarch64" = "arm64";
          }.${stdenv.hostPlatform.parsed.cpu.name}
        or (throw "no entry for cpu: ${stdenv.hostPlatform.parsed.cpu.name}"));
```

I think this way it's easier to read